src/assets/env.template.js

```diff
@@ -1,8 +0,0 @@
-(function (window) {
```

Comment:
Can you please revert these changes? I believe there was a rebase/merging issue with this branch.

Reply:
Done